src/data/dialogmotekandidat/types/dialogmoteunntakTypes.ts

```diff
@@ -7,6 +7,28 @@ export enum UnntakArsak {
   ARBEIDSFORHOLD_OPPHORT = "ARBEIDSFORHOLD_OPPHORT",
 }
 
+export enum CreateUnntakArsak {
```

Comment:
Kunne man delt det opp på en måte som gjør det enda mer tydelig at noe har skjedd her?:
`CreateUnntakArsak { ... } ` sånn som her, og så en `DepricatedUnntakArsak`, og så heller ha `UnntakArsak = CreateUnntakArsak & DepricatedUnntakArsak` eller noe sånt?

Reply:
union / intersection typer er ganske hårete med enums i typescript. Måtte eventuelt endret til å bruke vanlig string literal type:
eks: 
`type CreateUnntakArsak = "MEDISINSKE_GRUNNER" | "INNLEGGELSE_INSTITUSJON" | "FORVENTET_FRISKMELDING_INNEN_28UKER" | "DOKUMENTERT_TILTAK_FRISKMELDING"`
`type DeprecatedUnntakArsak = "FRISKMELDT" | "ARBEIDSFORHOLD_OPPHORT"`
`type UnntakArsak = CreateUnntakArsak | DeprecatedUnntakArsak`
Da bruker man bare stringen (eks `"FRISKMELDT"`) der hvor den brukes i stedet for `UnntakArsak.FRISKMELDT`, så det er litt mindre eksplisitt i bruk, men synes kanskje det er bedre enn å duplisere verdiene :thinking:

Reply:
Ja okey, da kan vi beholde det sånn 👍🏼

Reply:
Ble også litt forvirret av slik det er nå. Går dette?👇😊 Fikk ikke testet det, men får ingen kjørefeil i alle fall
```
export enum ValidUnntakArsak {
  MEDISINSKE_GRUNNER = "MEDISINSKE_GRUNNER",
  INNLEGGELSE_INSTITUSJON = "INNLEGGELSE_INSTITUSJON",
  FORVENTET_FRISKMELDING_INNEN_28UKER = "FORVENTET_FRISKMELDING_INNEN_28UKER",
  DOKUMENTERT_TILTAK_FRISKMELDING = "DOKUMENTERT_TILTAK_FRISKMELDING",
}

export enum DeprecatedUnntakArsak {
  FRISKMELDT = "FRISKMELDT",
  ARBEIDSFORHOLD_OPPHORT = "ARBEIDSFORHOLD_OPPHORT",
}

export type UnntakArsak = ValidUnntakArsak | DeprecatedUnntakArsak;
```

Reply:
Ser ut som det funka :heart_eyes: